### ISSUE TITLE
Gist is missing Region Landmark

### ISSUE THREAD

**Issue by aeroplanejane** [bbc/simorgh#9293]
**Describe the bug**
The Gist is missing the region landmark on an article page

> A region landmark is a perceivable section containing content that is relevant to a specific, author-specified purpose and sufficiently important that users will likely want to be able to navigate to the section easily and to have it listed in a summary of the page.

https://www.w3.org/TR/wai-aria-practices-1.1/examples/landmarks/region.html

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to '.https://www.test.bbc.com/arabic/articles/cl5m3453w36o.'
2. Navigate to Gist
3. Note that no region is announced 

**Expected behaviour**
Should meet acceptance criteria and screenreader UX here: https://zpl.io/29GOXxd

Given I use a screen reader
When I navigate to the Gist block on an article page
Then a region landmark is announced (e.g. Region)
And this must be announced with a label
And the label must be the same as the strong HTML element (i.e. the block title "At a glance")
And this landmark sits within the main landmark

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Checklist**
- [ ] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Additional context**
